lib/iphone-simulator.ts

```diff
@@ -146,6 +146,13 @@ export class iPhoneSimulator implements IiPhoneSimulator {
 			}
 		}
 
+		if (options.args) {
+			var args = options.args.trim().split(/\s+/);
```

Comment:
What happens when the user writes "a b" at the command prompt intending to give ios-sim a single parameter containing a space?

Do we need to take care of this case?